### PR TITLE
Refactor tag preview screens to new api

### PR DESCRIPTION
**Pull request by ajayesivan**
Part of Preview Screen Migration: #17288

Migrated all the tags preview screens to the new API.

Alongside the migration, I've also addressed two minor issues:
- Fixed token-tag Component: The `token-tag` component was not displaying the token image as expected.
- Fixed token-tag Usage in Permission Drawer Preview: updated the `token-tag` component prop usage.

### Testing Notes
- Test token-tag usage in permission drawer: quo2 preview -> drawers -> permission drawer
- Anywhere else we are using the token-tag component

status: ready

### CODE REVIEW

File: src/quo2/components/tags/token_tag.cljs [hunk 1]
```diff
@@ -33,7 +33,7 @@
    opts
    {
     :size :small/:big
-    :token-img-src :token-img-src
+    :img-src :img-src
```
Comment (J-Son89) [inline src/quo2/components/tags/token_tag.cljs]: 👍

**Review by J-Son89 — APPROVED** [review]
nice one @ajayesivan! 💪

**Review by smohamedjavid — APPROVED** [review]
🚀

File: src/status_im2/contexts/quo_preview/tags/permission_tag.cljs [hunk 2]
```diff
@@ -167,27 +165,23 @@
...
+       {:state                     state
+        :component-container-style {:margin-bottom 40}
+        :descriptor                descriptor}
+       (when @state
+         (for [{:keys [tokens]} community-tokens]
+           ^{:key tokens}
+           [rn/view
+            {:margin-top 20
+             :align-self :flex-end}
+            [permission-tag/tag
```
Comment (mohsen-ghafouri) [inline src/status_im2/contexts/quo_preview/tags/permission_tag.cljs]:
```suggestion
please use public quo2.core namespace
         [quo2.core :as quo]
         ```
            [quo/tag
```

File: src/status_im2/contexts/quo_preview/tags/status_tags.cljs [hunk 3]
```diff
@@ -1,10 +1,9 @@
 (ns status-im2.contexts.quo-preview.tags.status-tags
   (:require [quo2.components.tags.status-tags :as quo2]
```
Comment (mohsen-ghafouri) [inline src/status_im2/contexts/quo_preview/tags/status_tags.cljs]:
```suggestion
  (:require [quo2.core :as quo]
```

File: src/status_im2/contexts/quo_preview/tags/tag.cljs [hunk 4]
```diff
@@ -1,86 +1,49 @@
 (ns status-im2.contexts.quo-preview.tags.tag
-  (:require [react-native.core :as rn]
-            [quo2.foundations.colors :as colors]
-            [quo2.components.tags.tag :as tag]
-            [status-im.ui.components.react :as react]
+  (:require [quo2.components.tags.tag :as tag]
```
Comment (mohsen-ghafouri) [inline src/status_im2/contexts/quo_preview/tags/tag.cljs]:
```suggestion
  (:require [quo2.core :as quo]
```

File: src/status_im2/contexts/quo_preview/tags/tags.cljs [hunk 5]
```diff
@@ -1,50 +1,40 @@
 (ns status-im2.contexts.quo-preview.tags.tags
   (:require [quo2.components.tags.tags :as tags]
```
Comment (mohsen-ghafouri) [inline src/status_im2/contexts/quo_preview/tags/tags.cljs]:
```suggestion
  (:require [quo2.core :as quo]
```

File: src/status_im2/contexts/quo_preview/tags/token_tag.cljs [hunk 6]
```diff
@@ -43,17 +38,19 @@
...
         :descriptor descriptor}
-       [rn/view {:padding-bottom 150}
-        [rn/view
-         {:padding-vertical 60
-          :align-items      :center}
-         [quo2/token-tag
-          (merge @state
-                 {:token-img-src (if (= (get-in @state [:token]) "ETH") eth-token snt-token)})]]]])))
+       [rn/view {:style {:align-items :center}}
+        [quo2/token-tag
```
Comment (mohsen-ghafouri) [inline src/status_im2/contexts/quo_preview/tags/token_tag.cljs]: please use public quo2.core namespace
         [quo2.core :as quo]
         ```
            [quo/token-tag

**Review by mohsen-ghafouri — COMMENTED** [review]
Great job 🚀, i just left some comments about using the public `quo2.core` namespace.

**Review by jo-mut — APPROVED** [review]

**Comment by ajayesivan** [comment]
@mohsen-ghafouri Thanks for the review! I have made the suggested changes.

**Review by mohsen-ghafouri — APPROVED** [review]
Nice work 🚀

**Comment by churik** [comment]
@ajayesivan any clues where in app this component can be used?

**Comment by ajayesivan** [comment]
Hi @churik!
I fixed the tags/token-tag component not displaying the token image issue in this PR.

The token-tag component is being used in
- drawers/permission_drawers - only used in quo preview
- community/token-gating - being used in [token gated community home screens](https://www.figma.com/file/h9wo4GipgZURbqqr1vShFN/Communities-for-Mobile?type=design&node-id=8168-600795&mode=design&t=DYVw4qU5Va00RQrr-4)

**Comment by churik** [comment]
@ajayesivan component itself looks great on preview, but likely not used yet on the implementation (the same as in nightly, so I do not think it is something related to this PR)
IMAGE 2023-10-12 12:34:51

so I'd suggest merging this PR after passing the design review (cc @Francesca-G )

Thank you!

**Review by Francesca-G — COMMENTED** [review]
Here's the component design [review](https://www.figma.com/file/Tf5nfkYvpbnNCo4rKLK7lS/Feedback-for-Mobile?type=design&node-id=3415%3A214763&mode=design&t=WPUzLl4yreQwTwos-1) :)

**Comment by ajayesivan** [comment]
@Francesca-G I have fixed the component review issues. Could you please have another look? Thanks!

**Review by Francesca-G — APPROVED** [review]

**Pull request merged**